### PR TITLE
Fdispatch is now gone - no mor eneed for this flag.

### PR DESCRIPTION
**Pull request by baldersheim**
@bjorncs PR

### CODE REVIEW

File: config-model-api/src/main/java/com/yahoo/config/model/api/ModelContext.java [hunk 1]
```diff
@@ -54,8 +54,8 @@
         boolean isBootstrap();
         boolean isFirstTimeDeployment();
         boolean useDedicatedNodeForLogserver();
-        boolean useFdispatchByDefault();
-        boolean dispatchWithProtobuf();
+        boolean useFdispatchByDefault(); // TODO Remove once 7.110 is the oldest old config models
```
Comment (bjorncs) [inline config-model-api/src/main/java/com/yahoo/config/model/api/ModelContext.java]: Add default implementation returning `false` and `true` respectively.

File: config-model/src/main/java/com/yahoo/config/model/deploy/TestProperties.java [hunk 2]
```diff
@@ -60,8 +58,8 @@
     @Override public boolean isFirstTimeDeployment() { return isFirstTimeDeployment; }
     @Override public boolean useAdaptiveDispatch() { return useAdaptiveDispatch; }
     @Override public boolean useDedicatedNodeForLogserver() { return useDedicatedNodeForLogserver; }
-    @Override public boolean useFdispatchByDefault() { return useFdispatchByDefault; }
-    @Override public boolean dispatchWithProtobuf() { return dispatchWithProtobuf; }
+    @Override public boolean useFdispatchByDefault() { return false; }
```
Comment (bjorncs) [inline config-model/src/main/java/com/yahoo/config/model/deploy/TestProperties.java]: Remove implementations of `useFdispatchByDefault`/`dispatchWithProtobuf`.

File: configserver/src/main/java/com/yahoo/vespa/config/server/deploy/ModelContextImpl.java [hunk 3]
```diff
@@ -221,10 +215,10 @@ public String athenzDnsSuffix() {
         public boolean useDedicatedNodeForLogserver() { return useDedicatedNodeForLogserver; }
 
         @Override
-        public boolean useFdispatchByDefault() { return useFdispatchByDefault; }
+        public boolean useFdispatchByDefault() { return false; }
```
Comment (bjorncs) [inline configserver/src/main/java/com/yahoo/vespa/config/server/deploy/ModelContextImpl.java]: Remove implementations of `useFdispatchByDefault`/`dispatchWithProtobuf`.

**Review by bjorncs — CHANGES_REQUESTED** [review]
See comments

File: config-model-api/src/main/java/com/yahoo/config/model/api/ModelContext.java (hunk 1 above)
Comment (baldersheim) [reply config-model-api/src/main/java/com/yahoo/config/model/api/ModelContext.java]: That is not an API change ?

Comment (bratseth) [reply config-model-api/src/main/java/com/yahoo/config/model/api/ModelContext.java]: Not a public API

Comment (baldersheim) [reply config-model-api/src/main/java/com/yahoo/config/model/api/ModelContext.java]: Confirmed, with @hmusum that is was fine.

File: configserver/src/main/java/com/yahoo/vespa/config/server/deploy/ModelContextImpl.java (hunk 3 above)
Comment (baldersheim) [reply configserver/src/main/java/com/yahoo/vespa/config/server/deploy/ModelContextImpl.java]: Done

File: config-model/src/main/java/com/yahoo/config/model/deploy/TestProperties.java (hunk 2 above)
Comment (baldersheim) [reply config-model/src/main/java/com/yahoo/config/model/deploy/TestProperties.java]: Done

File: config-model-api/src/main/java/com/yahoo/config/model/api/ModelContext.java (hunk 1 above)
Comment (baldersheim) [reply config-model-api/src/main/java/com/yahoo/config/model/api/ModelContext.java]: Done

**Review by bjorncs — COMMENTED** [review]
👍

**Pull request merged**